tests/gocase/unit/type/hash/hash_test.go

```diff
@@ -21,67 +21,682 @@ package hash
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
 	"testing"
+	"time"
 
 	"github.com/apache/incubator-kvrocks/tests/gocase/util"
 	"github.com/stretchr/testify/require"
 )
 
+func getKeys(hash map[string]string) []string {
+	r := make([]string, 0)
+	for key := range hash {
+		r = append(r, key)
+	}
+	return r
+}
+
+func getVals(hash map[string]string) []string {
+	r := make([]string, 0)
+	for _, val := range hash {
+		r = append(r, val)
+	}
+	return r
+}
+
+func getAllSorted(hash map[string]string) map[string]string {
```

Comment:
I think it won't sort anything since the Go's map is unordered, so what this function do is copy the map without any sort.

Reply:
thranks for your comment , i will check it